src/04_printing.py

```diff
@@ -11,7 +11,9 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("printf operator way: " + 'x is %(x)d, y is %(y).2f, z is "%(z)s"' % {"x" : x, "y" : round(y, 2), "z" : z})
```

Comment:
All these outputs look great! Just to show you a very clean option for this first method:
`print('x is %d, y is %.2f, z is "%s"' % (x, y, z))`